pkg/types/baremetal/validation/platform.go

```diff
@@ -17,6 +17,13 @@ func validateIPinMachineCIDR(vip string, n *types.Networking) error {
 	return nil
 }
 
+func validateIPNotinMachineCIDR(ip string, n *types.Networking) error {
+	if n.MachineCIDR.Contains(net.ParseIP(ip)) {
+		return fmt.Errorf("the IP is not expected to be in %s subnet", n.MachineCIDR.String())
```

Comment:
Can we strengthen the wording on this fatal error?  Maybe

```go
return fmt.Errorf("%s must not be in %s", ip, n.MachineCIDR.String())
```

Reply:
Done